README.md

```diff
@@ -38,8 +39,8 @@ Once you've set up a local copy of the repo, you can run the server in various w
 Build from the local repo and run in foreground:
 
 ```
-docker build -t translation-server .
-docker run -ti -p 1969:1969 --rm translation-server
+docker build --tag translation-server .
+docker run --tty --interactive --publish 1969:1969 --rm translation-server
```

Comment:
I don't think there's any need to spell all these out. These are all basic flags, and what people should be used to. Even Docker's own documentation uses the abbreviations:

https://docs.docker.com/engine/reference/commandline/build/#tag-an-image--t

Reply:
I think it helps with readability, I myself end up forgetting the short args. And helps with users who are not familiar with Docker to understand what the commands do, without us having to write comments in README. But will revert this if you prefer.

Reply:
Yes, please revert. It's not our job to teach people Docker, and the abbreviations are more representative of standard Docker usage.